Sources/Testing/Expectations/ExpectationChecking+Macro.swift

```diff
@@ -1148,18 +1148,18 @@ public func __checkClosureCall<R>(
 @_spi(Experimental)
 public func __checkClosureCall(
   identifiedBy exitTestID: (UInt64, UInt64),
-  exitsWith expectedExitCondition: ExitCondition,
-  observing observedValues: [any PartialKeyPath<ExitTestArtifacts> & Sendable],
+  exitsWith requirement: ExitTest.Condition,
```

Comment:
This is one of the situations I was thinking of in another comment about using the term `condition` most of the time.

Also, I see other decls still use the argument name `expectedExitCondition`